election-dapp/src/index.css

```diff
@@ -0,0 +1,50 @@
+
+
+.Menu{
```

Comment:
indentation required here(use Ctrl+shift+i) to indent the file content in VS code